install_config/topics/configuring_for_gce.adoc

```diff
@@ -0,0 +1,197 @@
+////
+Module included in the following assemblies:
+
+install_config/configuring_gce.adoc
+////
+
+[id='gce-configuring-masters-ansible_{context}']
+= Option 1: Configuring {product-title} for GCP using Ansible
+
+You can configure {product-title} for Google Compute Platform (GCP) by
+modifying the
+xref:../install/configuring_inventory_file.adoc#configuring-ansible[Ansible
+inventory file] at installation time or after installation.
+
+.Procedure
+. At minimum, you must define the `openshift_cloudprovider_kind`, `openshift_gcp_project` and
+`openshift_gcp_prefix` parameters. For
+xref:../configuring_gce.adoc#gce-load-balancer_configuring-for-GCE[running load balancer services
+using GCP], the Compute Engine VM node instances require the `ocp` suffix. For example, if the value of the
+`openshift_gcp_prefix` parameter is set to `mycluster`, you must tag the nodes
+with `myclusterocp`. See
+link:https://cloud.google.com/vpc/docs/add-remove-network-tags[Adding and
+Removing Network Tags] for more information on how to add network tags to
+Compute Engine VM instances.
++
+Add the following to the Ansible inventory file at installation to configure
+your {product-title} environment for GCP:
++
+----
+[OSEv3:vars]
+openshift_cloudprovider_kind=gce
+openshift_gcp_project=<projectid> <1>
+openshift_gcp_prefix=<uid> <2>
+openshift_gcp_multizone=False <3>
+openshift_gcp_network_name=<network name> <4>
+----
+<1> Set to the GCP project ID where the existing instances are running.
+<2> Set to a unique string to identify each {product-title} cluster.
```

Comment:
The GCP project ID is generated from the GCP project name you enter when you create the project in the Google Cloud Platform Console. Certain words are restricted from use in project IDs. If you use restricted words in the project name, such as google or ssl, the generated project ID will not include these words. This will not affect the project name.

The project number and project ID are unique across Google Cloud Platform. If another user owns a project ID for their project, you won't be able to use the same project ID.

(as per https://cloud.google.com/resource-manager/docs/creating-managing-projects )

Reply:
That's good to know. @bfallonf, do you want to include some of this context?